docs/authentication/embedded-sdk-add-passkeys.mdx

```diff
@@ -48,10 +48,18 @@ If your application is using the [NextAuth](https://next-auth.js.org) provider (
 
 - [x] **Authenticator Config**: Embedded SDK <mark>If they are required to point the Invoke URL to their application, shouldn't we say that it's a prerequisite? As mentioned below, the Invoke URL field is required and they can't leave this blank when creating their app in the Admin Console.</mark>
```

Comment:
Yes, should we include all required Authenticator Config options in this section?

Reply:
It looks like Embedded has it's own file and Hosted has it's own file, so we can probably focus on the respective ones in each file.

Reply:
Right, I meant to say should we include Invocation Type, Invocation URL and Trusted Origins (for Embedded) here (and Authentication Profile and Trusted Origins for Hosted Web there)